src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+			</li>
+		</ul>
+	</div>
+);
+
+//ToDo Update Object Links
+const update1904: ReactNode = (
+	<div className="paragraph">
+		<ul>
+			<li>
+				<p>AddInData (Feature Preview) - Groups are now optional for AddInData objects, currently in Feature preview. Previously, groups were a required property for the AddInData object. This limited the potential usage of AddInData as there are situations where data should be available to all users regardless of scope, and some users were not able to access data when they belonged to groups outside the data’s scope. Removing this restriction means any user is now allowed to get an AddInData object if no group is specified for the object.</p>
+			</li>
+			<li>
+				<p>API.cs (nuget only) - Now implements IApi interface. This allows for simpler unit testing of integration code using mocks.</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#BinaryData">BinaryDataType</a> - Added <code className="small-code-sample">SoftwareVersionFull</code></p>
+			</li>
+			<li>
+				<p>Calculated Engine Hours Search - With a custom setting (<code className="small-code-sample">ENABLEENGINEHOURSINTERPOLATION</code>) applied to your database, DiagnosticEngineHoursAdjustmentId will now be interpolated using trips and DiagnosticIgnitionId values when a search includes a from/toDate value(s) to provide exact values. To apply this custom setting to your database, please reach out to Geotab support. This will become the default behavior in v5.7.2001.</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DatabaseExistsAsync">DatabaseExists</a> fixed to include databases existing in other federations.</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#FuelTransactionProductType">FuelTransactionProductType</a> - Added <code className="small-code-sample">Hydrogen</code> and <code className="small-code-sample">DieselExhaustFluid</code>.</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#FuelTransactionProvider">FuelTransactionProvider</a> - Added <code className="small-code-sample">GFN</code>.</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#HosRuleSet">HosRuleSet</a> - Added <code className="small-code-sample">HosRuleSetCanadaCycleOneTeam</code> and <code className="small-code-sample">HosRuleSetCanadaCycleTwoTeam</code>.</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#LoginResult">LoginResult</a> - Removed unsupported legacy property <code className="small-code-sample">SecurityToken</code>. This property duplicated the supported property <code className="small-code-sample">Credentials</code>. It was previously maintained for compatibility with MyGeotab Web Server 5.6.1 which is no longer supported.</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#Rule">Rule</a> - Fix, don’t allow adding Rules without Conditions.</p>
```

Comment:
The character U+2019 "’" could be confused with the ASCII character U+0060 "`", which is more common in source code